src/helpers.js

```diff
@@ -23,14 +23,10 @@ export function getFindDOMNode() {
 }
 
 export function getFirstOrderedNodeType() {
-  if (XPathResult) {
-    return XPathResult.FIRST_ORDERED_NODE_TYPE;
+  const xpathResult = global && global.XPathResult || window && window.XPathResult;
```

Comment:
This is the gist of the fix: explicitly check the `global` object instead of depending a-priori on a global variable.